lib/middleware/auth.ts

```diff
@@ -0,0 +1,35 @@
+/// <reference path='../../typings/tsd.d.ts' />
+
+// Handles client auth operations.
+// Currently we only have authorization (e.g. Identity management), associating the identities with
+// client certs.
+
+import blpapi = require('blpapi');
+import _ = require('lodash');
+import restify = require('restify');
+import Interface = require('../interface');
+import Map = require('../util/map');
+
+var identityMap: Map<blpapi.Identity> = new Map<blpapi.Identity>();
```

Comment:
How about `s_identityMap` to be consistent with the C++ naming convention for static variables?